### PR TITLE
local dev on macos

### PR DESCRIPTION
**Pull request by philbrookes**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -87,6 +87,16 @@ $ make e2e
...
 
+### Local development on a mac
+MacOS does not default to allowing you to communicate with ports running in docker containers from the host machine. To bypass this limitation we can use `docker-mac-net-connect`:
+```
+# Install via Homebrew
+$ brew install chipmk/tap/docker-mac-net-connect
+
+# Run the service and register it to launch at boot
+$ sudo brew services start chipmk/tap/docker-mac-net-connect
+```
```
Comment (mikenairn) [inline README.md:98]: I have no way to verify this, so someone else should probably do that, but does this allow you to curl the generated endpoints from your local machine and have it served by the local kind cluster?

Comment (philbrookes) [reply README.md:98]: I think @maleck13 is going to give it a go.

Comment (philbrookes) [reply README.md:98]: missed the question. Yeah it does, I can also use my browser on the DNS that's created in route53.

**Review by maleck13 — APPROVED** [review]
I think it would be good to call out that you need an AWS account, a domain and route53 also

File: utils/local-setup.sh [hunk 2]
```diff
@@ -29,6 +31,9 @@ while getopts ":c:" arg; do
 done
 shift $((OPTIND-1))
 
+if [[ "${OSTYPE}" =~ ^darwin.* ]]; then
```
Comment (maleck13) [inline utils/local-setup.sh:34]: I think I would make overridable so its not forced on you?

Comment (philbrookes) [reply utils/local-setup.sh:34]: I did consider that,  ok I'll add it

**Comment by maleck13** [comment]
I got 

```
./utils/local-setup.sh -c 2
Error: Formula `docker-mac-net-connect` is not installed.

```

**Comment by maleck13** [comment]
worked as expected

**Pull request merged**